py_gtfs_rt_ingestion/py_gtfs_rt_ingestion/logging_utils.py

```diff
@@ -0,0 +1,72 @@
+import logging
+import time
+import uuid
+
+from typing import Optional, Union
+
+MdValues = Union[str, int, float]
+
+
+class ProcessLogger:
+    """
+    Class to help with logging events that happen inside of a function.
+    """
+
+    def __init__(
+        self, process_name: str, **metadata: Optional[MdValues]
```

Comment:
I don't think you need `Optional` on this `**metadata` type hint, I believe `metadata` is just an empty dictionary in the case of no keyword arguments being passed in